xc7/litex_demo/Makefile

```diff
@@ -0,0 +1,25 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
The `xc7/litex_demo` directory should have a `README.rst` file

Reply:
added